testssl.sh

```diff
@@ -19783,6 +19788,9 @@ parse_cmd_line() {
                --ssl_native|--ssl-native)
                     SSL_NATIVE=true
                     ;;
+               --basicauth|--basicauth=*)
+                   BASICAUTH="$(parse_opt_equal_sign "$1" "$2")"
+                   ;;
```

Comment:
You would need to add ``[[ $? -eq 0 ]] && shift`` here, otherwise the value of  <user:pass> will be treated as another argument to testssl.sh

Reply:
> Moreover a line in `help()` is missing
The line in `help()` is included in my PR (line 17167). I currently don't get what is missing there. Could you please be more specific on this?